addOns/retire/src/test/java/org/zaproxy/addon/retire/RetireScanRuleUnitTest.java

```diff
@@ -220,13 +224,18 @@ void shouldRaiseAlertOnHashOfVulnerableContent() {
         assertEquals(1, alertsRaised.size());
         assertThat(alertsRaised.get(0).getRisk(), is(equalTo(Alert.RISK_LOW)));
         assertEquals(
-                "CVE-XXXX-XXX2\n"
-                        + "CVE-XXXX-XXX1\n"
-                        + "CVE-XXXX-XXX0\n"
-                        + "The library matched the known vulnerable hash e19cea51d7542303f6e8949a0ae27dd3509ea566.",
+                """
+                The identified library hash-test-entry, version 0.0.1 is vulnerable.
+                CVE-XXXX-XXX2
+                CVE-XXXX-XXX1
+                CVE-XXXX-XXX0
+                The library matched the known vulnerable hash e19cea51d7542303f6e8949a0ae27dd3509ea566.
+                http://example.com/hash-test-entry
+                http://example.com/hash-test-entry2
+                """,
                 alertsRaised.get(0).getOtherInfo());
         assertEquals(
-                "http://example.com/hash-test-entry\nhttp://example.com/hash-test-entry2\n",
+                "https://owasp.org/Top10/A06_2021-Vulnerable_and_Outdated_Components/",
                 alertsRaised.get(0).getReference());
```

Comment:
Extracting a method for this would be better now that it's common to all (and can be added to the example alert too).